src/main/java/net/wasdev/wlp/common/plugins/util/ServerFeatureUtil.java

```diff
@@ -0,0 +1,314 @@
+package net.wasdev.wlp.common.plugins.util;
```

Comment:
Add copyright

Reply:
fixed in https://github.com/WASdev/ci.common/pull/52